src/Skia/Avalonia.Skia/FontManagerImpl.cs

```diff
@@ -30,53 +30,86 @@ public IEnumerable<string> GetInstalledFontFamilyNames(bool checkForUpdates = fa
             return _skFontManager.FontFamilies;
         }
 
-        public Typeface GetTypeface(FontFamily fontFamily, FontWeight fontWeight, FontStyle fontStyle)
-        {
-            return TypefaceCache.Get(fontFamily.Name, fontWeight, fontStyle).Typeface;
-        }
+        [ThreadStatic] private static string[] s_languageTagBuffer;
 
-        public Typeface MatchCharacter(int codepoint, FontWeight fontWeight = default, FontStyle fontStyle = default,
-            FontFamily fontFamily = null, CultureInfo culture = null)
+        public bool TryMatchCharacter(int codepoint, FontWeight fontWeight, FontStyle fontStyle,
+            FontFamily fontFamily, CultureInfo culture, out FontKey fontKey)
         {
-            var fontFamilyName = FontFamily.Default.Name;
-
             if (culture == null)
             {
                 culture = CultureInfo.CurrentUICulture;
             }
 
+            if (s_languageTagBuffer == null)
```

Comment:
@grokys we probably need a different naming convention for `[ThreadStatic]` fields.

Reply:
Perhaps yeah, do you know what anyone else uses?

Reply:
`s_tlsFieldName` might be an option.

Reply:
CoreFX uses `t_fieldName`